### DISCUSSION TITLE
Add URL text without automatic hyperlink being created

### DISCUSSION THREAD

**Discussion by JoelMTaylor** [retypeapp/retype#369] [Q&A] [answered]
Hi, I just wanted to quickly ask how one could prevent a URL from appearing with a hyperlink? That is, I want the URL text (https://www..wikipedia.org/wiki/Main_Page) without the hyperlink automatically being created.

P.S. Since this was a minor question, I tried asking in the Retype chat ('run on crisp'). However, it kept informing me that "Failed to send. Retry?"

**Comment by geoffreymcgill** [comment, marked as answer]
Currently, there is no way to disable the automatic parsing of hyperlinks.

How about wrapping the link in back ticks, like an inline code sample?

```
That is, I want the URL text (`https://www..wikipedia.org/wiki/Main_Page`) without the hyperlink automatically being created.
```

**Reply by JoelMTaylor** [reply]
Okay, no problem. I appreciate the idea.
